### ISSUE TITLE
Generating themes without linking

### ISSUE THREAD

**Issue by fushiii** [danth/stylix#156]

**Comment by dwarfmaster** [comment]
It may make sense to include the `themeFile` option for the modules where we generate it ourselves, but for the wezterm module we use the home-manager module to set the values (and looking at the [wezterm module](https://github.com/danth/stylix/blob/master/modules/wezterm/hm.nix) it seems we need to add some lua to the main config file).
So maybe we can support this on a best effort basis, but since we're using hm we're more or less under the assumption that we handle fully the configuration of the packages we theme.

**Comment by dwarfmaster** [comment]
For your specific situation I can think of two solutions:
- Add `xdg.configFile."wezterm/wezterm.lua".enable = lib.mkForce false` to disable the generation of the wezterm.lua file while still enabling wezterm in home-manager
- Build your config without activationg it, and copy the file `result/home-files/.config/wezterm/colors/stylix.toml` to the right place.

**Comment by fushiii** [comment]
> It may make sense to include the `themeFile` option for the modules where we generate it ourselves, but for the wezterm module we use the home-manager module to set the values (and looking at the [wezterm module](https://github.com/danth/stylix/blob/master/modules/wezterm/hm.nix) it seems we need to add some lua to the main config file). So maybe we can support this on a best effort basis, but since we're using hm we're more or less under the assumption that we handle fully the configuration of the packages we theme.

That does make sense. Something along those lines would likely exceed the functionality that Stylix aims to achiev

**Comment by danth** [comment]
It sounds like you might be better off with a standard base16 builder rather than Stylix? That would generate all of the theme files and leave them for you to put into place by hand.

**Comment by fushiii** [comment]
> It sounds like you might be better off with a standard base16 builder rather than Stylix? That would generate all of the theme files and leave them for you to put into place by hand.

That's true. Although i haven't found any of them yet, it sounds just like what i need. Could you recommend me any?

**Comment by danth** [comment]
[Flavours](https://github.com/Misterio77/flavours) looks relevant and is packaged in Nixpkgs
